src/api/Alert/types.ts

```diff
@@ -317,6 +321,11 @@ export interface ICaseColumn {
   columnTitle: string;
 }
 
+export interface ModelData {
```

Comment:
Since `models` are actually referred by alert queries in the restClient and server code bases, how about we rename  the interface to `alertQueryData`